catkit/hardware/iris_ao/segmented_dm_command.py

```diff
@@ -338,6 +338,9 @@ def to_ini(self, filename, out_dir=''):
         :param filename: str, name of ini file to be written out
         :param out_dir: str, name of directory where ini file will be saved
         """
+        if not filename.endswith((".ini", ".ini")):
```

Comment:
@ivalaginja Dit you mean to duplicate the ``".ini"``?

Reply:
That's how I found it here:
https://github.com/spacetelescope/catkit/blob/a7740f7fdf4606a1b995214ee23ac798de9c04c0/catkit/util.py#L131

Reply:
Note that they're not identical ``(".fit", ".fits")``.

Reply:
This just needs to be ``filename.endswith(".ini")``

Reply:
fixed